src/dictionary.c

```diff
@@ -191,6 +191,15 @@ static void SpellCheckDictAuxSave(RedisModuleIO *rdb, int when) {
   dictReleaseIterator(iter);
 }
 
+static void SpellCheckDictAuxSave2(RedisModuleIO *rdb, int when) {
+  if (when == REDISMODULE_AUX_BEFORE_RDB) {
```

Comment:
Consider removing (checked in SpellCheckDictAuxSave)

Reply:
OK. Done.